### PR TITLE
Sentinel 2 Cloud Dataset

### PR DESCRIPTION
**Pull request by kokoedwin**

### CODE REVIEW

**Comment by andreaschandra** [comment]
review @rubentea16

File: earthvision/datasets/sentinel2cloud.py [hunk 1]
```diff
@@ -1,2 +1,108 @@
...
+        sample = (image, mask)
+
+        return sample
+
+    def __len__(self):
+        return len(self.img_labels)
+
+    def __iter__(self):
+        for index in range(self.__len__()):
+            yield self.__getitem__(index)
```
Comment (rubentea16) [inline earthvision/datasets/sentinel2cloud.py:60]: `__iter__` => unnecessary

**Review by rubentea16 — CHANGES_REQUESTED** [review]

**Comment by rubentea16** [comment]
@kokoedwin @FendiSuryadinata570

File: earthvision/datasets/sentinel2cloud.py [hunk 2]
```diff
@@ -1,2 +1,123 @@
...
+    mirrors = "https://zenodo.org/record/4172871/files/"
+    resources = "subscenes.zip"
+    mask_resources = "masks.zip"
+
+    def __init__(self,
+                 root: str):
+        
+        self.root = root
+
+        if os.path.exists(self.root):
```
Comment (rubentea16) [inline earthvision/datasets/sentinel2cloud.py:45]: just use `if not os.path.exists(self.root): os.makedirs(self.root)`

**Review by rubentea16 — APPROVED** [review]

**Comment by rubentea16** [comment]
silahkan di merge @kokoedwin @FendiSuryadinata570

**Pull request merged**